xslt/elements/stylesheet.json

```diff
@@ -0,0 +1,250 @@
+{
+  "xslt": {
+    "elements": {
+      "stylesheet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/XSLT/Element/stylesheet",
```

Comment:
This link is pointing to a 404 - suspect the `/Element/` is not required.

Reply:
See #2027

Reply:
**Update:** I’ve now moved the page to the new location.